webrecorder/browsertrix
# Filter and sort crawl templates

(https://github.com/webrecorder/browsertrix-cloud/issues/153) Filters, sorts and enables searching crawl templates by name. Crawls list and browser profiles list tweaked for UI consistency.

### Manual testing
Go to Crawl Templates and test that the following works as expected:
- "Search by name" input
- Filter by "Scheduled" or "No schedule", or show all with "All"
- Sort by newest/oldest crawl template and newest/oldest crawl

### Screenshots
**New control bar:**
<img width="1027" alt="Screen Shot 2022-04-18 at 4 33 12 PM" src="https://user-images.githubusercontent.com/4672952/163893734-3340cea3-0517-4216-b7f3-ffda33c7347f.png">

**Sort options:**
<img width="213" alt="Screen Shot 2022-04-18 at 4 33 21 PM" src="https://user-images.githubusercontent.com/4672952/163893765-6ee6ed0d-4306-4c66-b39c-feafe71d9bd7.png">

**Filter by name:**
<img width="1032" alt="Screen Shot 2022-04-18 at 4 41 14 PM" src="https://user-images.githubusercontent.com/4672952/163893776-cb978256-d7a2-4e9b-ad24-a02268ce4515.png">
